dbcmetrics/dbcm_logic/dbcplannedmaintenance.py

```diff
@@ -0,0 +1,150 @@
+        #                                            platform name          window_start_date      window_end_date   window_start_date  window_end_date       ...
+        
+        for platform in self.PLATFORMS:
+            try:
+                platform_name = platform['name']
+                logging.info(f"platform_name = {platform_name}, plattform_url = {platform['url']}")
+
+                url = platform['url'] + "/api/v1/schedules"
+
+                headers = {
+                    "accept": "application/json"
+                }
+
+                response = json.loads(requests.get(url, headers=headers).text)
+
+                platform_windows = []
+                # Several maintance_entry may exist in response['data'], default is empty List
+                for maintance_entry in response['data']:
+                    # load and parse data from response
+
+                    try:
+                        logging.info(f"Found Mainenance entry in {platform_name}, title: {maintance_entry['name']}, message: {maintance_entry['message']}")
+                    except Exception as e:
+                        # Triggered if message field is empty
+                        # It is not a requiered field, so code will continue for this entry
+                        logging.warning(f"Couldn't load maintenance entry: title or message on {platform_name} for entry: {maintance_entry}")
+                        logging.exception(f"\t {e}")
+
+
+                    try:
+                        platform_window_start = self.parse_string_to_datetime(maintance_entry['scheduled_at'])
+                    except Exception as e:
+                        # Triggered if field for start of platform maintenance window is empty
+                        logging.warning(f"Couldn't load maintenance entry: platform_window_start on {platform_name} for entry: {maintance_entry}")
+                        logging.exception(f"\t {e}")
+                        continue
+
+                    try:
+                        message_string: str = maintance_entry['message']
+                        stunde_result = re.findall(r"([\d]+)\s*h", message_string) # returns e.g.: 2, 10, 1 as String in a List
+                        start_to_end_window_hours: int = int(stunde_result[0])
+                        succsessfully_parsed_hours: bool = True
+                    except Exception as e:
+                        logging.warning(f"\tCouldn't parse hours of end of maintenance window, using default hour offset (0 hours).")
+                        succsessfully_parsed_hours: bool = False
+                        start_to_end_window_hours: int = 0
+                    
+                    try:
+                        minute_result = re.findall(r'([\d]+)\s*m', message_string) # returns e.g.: 2, 10, 40 as String in a List
+                        start_to_end_window_min: int = int(minute_result[0])
+                    except Exception as e:
```

Comment:
Do I see it right that, when only hours or minutes are defined, a warning about not parsable maintenance window is produced?